python_modules/libraries/dagster-dg/dagster_dg/cli/init.py

```diff
@@ -38,19 +38,11 @@ def init_command(
     project_name: Optional[str],
     **global_options: object,
 ):
-    """Initialize a new Dagster workspace and a first project within that workspace.
+    """Initialize a new Dagster project or workspace.
 
-    The scaffolded workspace folder has the following structure:
-
-    \b
-    ├── dagster-workspace
-    │   ├── projects
-    |   |   └── <Dagster projects go here>
-    |   ├── libraries
-    |   |   └── <Shared packages go here>
-    │   └── pyproject.toml
-
-    """  # noqa: D301
+    By default, only a standalone project is created. A workspace can also be created using
+    the --workspace-name option.
+    """
```

Comment:
I think it would be helpul to retain the existing example structure and then to also have the project structure.